rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java

```diff
@@ -42,7 +42,9 @@ public RawMonitor(String outputName, RVMonitorSpec rvmSpec,
                     rvmSpec.getEvents(), coenableSet);
         }
 
-        monitorDeclaration = new UserJavaCode(rvmSpec.getDeclarationsStr());
+        String declStr = rvmSpec.getDeclarationsStr();
+        declStr = declStr.replaceAll("__CONSTRUCTOR", monitorName.toString());
```

Comment:
Please document in comments what use case this is meant to support. Why do we need this?

Reply:
done

Reply:
LGTM. Thanks.

Reply:
Can you approve the PR?

Reply:
Done.